Services/GEV/CourseSearch/classes/class.gevCourseSearch.php

```diff
@@ -134,6 +134,19 @@ public function getPotentiallyBookableCourseIds($a_search_options) {
 				" AND edu_program.value LIKE ".$this->gDB->quote("%".$a_search_options["edu_program"]."%", "text")."\n";
 		}
 
+		if (array_key_exists("cat", $a_search_options) && $a_search_options["cat"][0] != "") {
+			$categorie_field_id = $this->gev_set->getAMDFieldId(gevSettings::CRS_AMD_TOPIC);
+
+			$additional_join .=
+				" LEFT JOIN adv_md_values_text cat\n".
+				"   ON cs.obj_id = cat.obj_id\n".
+				"   AND cat.field_id = ".$this->gDB->quote($categorie_field_id, "integer")."\n";
+			$additional_where .=
+				" AND (cat.value LIKE ".$this->gDB->quote("%".$a_search_options["cat"][0]."%")."\n".
+				" OR cat.value LIKE ".$this->gDB->quote("%".$a_search_options["cat"][1]."%")."\n".
+				" OR cat.value LIKE ".$this->gDB->quote("%".$a_search_options["cat"][2]."%").")\n";
+		}
+
```

Comment:
Warum bastelst das nicht in eine Schleife? Das würde es vereinfachen, wenn die GEV da wieder irgendwelche Dinge ändern will.

Reply:
Done